src/twitch.py

```diff
@@ -142,14 +145,17 @@ async def bgdailii(ctx):
 async def goodBot(ctx):
     await ctx.send('MrDestructoid Just doing my job MrDestructoid')
 
-@twitchBot.command(name='wall_lii')
-async def wall_lii(ctx):
-    await ctx.send('HeyGuys I\'m a bot that checks the BG leaderboard to get data about player ranks and daily MMR fluctuations. I reset daily at Midnight CA time. Try using !bgrank [name] and !bgdaily [name] and !yesterday [name]. Also try !buddy [hero] and !goldenbuddy [hero] for buddy info')
-
-@twitchBot.command(name='help')
+@twitchBot.command(name='help', aliases=['wall_lii'])
 async def help(ctx):
     await ctx.send('HeyGuys I\'m a bot that checks the BG leaderboard to get data about player ranks and daily MMR fluctuations. I reset daily at Midnight CA time. Try using !bgrank [name] and !bgdaily [name] and !yesterday [name]. Also try !buddy [hero] and !goldenbuddy [hero] for buddy info')
 
+@twitchBot.event()
```

Comment:
Traceback (most recent call last):
  File "src/twitch.py", line 152, in <module>
    @twitchBot.event()
TypeError: event() missing 1 required positional argument: 'func'

Reply:
that's weird I took that from the documentation https://twitchio.readthedocs.io/en/latest/exts/commands.html#twitchio.ext.commands.Bot.event_error

Reply:
I version locked twitchio to 1.2.3 and the latest version is far ahead of that now so it might not be supported. I will try to update to the latest version and fix anything that is broken because of that and then implement that function.